qiskit_nature/second_q/hamiltonians/electronic_energy.py

```diff
@@ -60,7 +61,10 @@ def __init__(
             nuclear_repulsion_energy: the optional nuclear repulsion energy.
             reference_energy: an optional reference energy (such as the HF energy).
         """
-        super().__init__(self.__class__.__name__, electronic_integrals, shift=energy_shift)
+        self._electronic_integrals: dict[ElectronicBasis, dict[int, ElectronicIntegrals]] = {}
+        for integral in electronic_integrals:
+            self.add_electronic_integral(integral)
+        self._shift = energy_shift or {}
         self._nuclear_repulsion_energy = nuclear_repulsion_energy
         self._reference_energy = reference_energy
```

Comment:
This will also be further changed in the upcoming refactoring, at which point only a single-basis `PolynomialTensor` will be stored internally here :+1: